project/frontend/src/pages/MatchResultsPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function MatchResultsPage(props) {
+  // Form state is in `props.location.state`.
+
+  return <p>Results page content goes here.</p>;
+}
+
+export default MatchResultsPage;
```

Comment:
what would happen if you changed this back to MatchResults.js ?

Reply:
Not sure what you mean. I just thing the name `MatchResultsPage` is better semantically because we have `HomePage` and will presumable have more pages like `UserSettingPage` or `SwipeMatchPage` and whatnot.

Reply:
gotchaaa. I was just wondering if you could change it back to its original name since we're most likely going to have some conflicts and didn't know we had that naming convention!